h/templates/accounts/notifications.html.jinja2

```diff
@@ -1,13 +1,23 @@
 {% extends "h:templates/layouts/account.html.jinja2" %}
 
+{% if not feature('activity_pages') %}
+{% set uses_legacy_styles = True %}
+{% endif %}
+
 {% set page_route = 'account_notifications' %}
 {% set page_title = 'Notifications' %}
 
 {% block page_content %}
-  <div class="form-vertical">
-    <h2 class="form-heading">
-      <span>{% trans %}Update your notification settings{% endtrans %}</span>
-    </h2>
-    {{ form }}
-  </div>
+  {% if feature('activity_pages') %}
+    <div class="form content">
```

Comment:
Yes, that would be fine. Whether to use say `form__nav` vs. `form-nav` depends on whether the tabs are something that could be re-used on its own or whether it only makes sense in the context of a parent `form`. https://en.bem.info/methodology/faq/#a-block-or-an-element-when-should-i-use-which explains this a bit more.

The idea is to make it clear from the class name whether a class can be used on its own or whether it needs to live inside some other block.